packages/engine/tests/units/mod.rs

```diff
@@ -0,0 +1,4 @@
+mod experiment;
```

Comment:
Perhaps we want to add a module docstring here just to explain what we mean by `units`. (As in, these _are_ integration tests but test discrete functionalities in the simulations somewhat similar to a unit-testing philosophy)

Also not a massive fan of non-`mod.rs` files declaring new modules as it makes the hierarchy difficult to parse without an IDE, but that's not a big deal.

Reply:
> Perhaps we want to add a module docstring here just to explain what we mean by `units`. (As in, these are integration tests but test discrete functionalities in the simulations somewhat similar to a unit-testing philosophy)

Will add one

> Also not a massive fan of non-`mod.rs` files declaring new modules as it makes the hierarchy difficult to parse without an IDE, but that's not a big deal.

Sadly this is not possible without duplicating the `integration` module. `cargo test` does not read `tests/integration/` as a library which then could define a `tests/integrations/units/` module. I'd prefer it that way, but sadly it's not working. (If I'm wrong, please correct me, this also annoys me, too).

Do you have a suggestion for a name other than `units/` so it's not confusing when only looking at `tests/`?